docs/reference/logging/overview.md

```diff
@@ -107,6 +111,14 @@ the following named facilities:
 * `local6`
 * `local7`
 
+`syslog-tls-ca-cert` specifies the absolute path in the host to the Trust certs signed by the CA. This option is ignored if the address protocol is not `tcp+tls`.
```

Comment:
minor nits;
s/Trust/trust/
s/certs/certificates/


also, can you wrap your lines to 80 chars?